FRINXio/frinx-frontend
# FD-678 workflow name filter fix

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-666` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |


IMPORTANT:
first check this https://github.com/FRINXio/frinx-frontend/pull/1561

HOW TO TEST:

test with https://github.com/FRINXio/frinx-graphql-proxy/pull/89

you should be able to see workflows once you clear workflow name in filter.https://frinxhelpdesk.atlassian.net/browse/FD-678 (11.)
